Modules/ThirdParty/Expat/src/expat/itk_expat_mangle.h

```diff
@@ -97,9 +96,27 @@ nm libITKEXPAT-4.9.a | grep " [TRD] " |  awk '{ print $3 }' | sort | awk '{ prin
 #define XML_SetUnparsedEntityDeclHandler itk_expat_XML_SetUnparsedEntityDeclHandler
 #define XML_SetUserData itk_expat_XML_SetUserData
 #define XML_SetXmlDeclHandler itk_expat_XML_SetXmlDeclHandler
-#define XmlSizeOfUnknownEncoding itk_expat_XmlSizeOfUnknownEncoding
+#define XML_StopParser itk_expat_XML_StopParser
+#define XML_UseForeignDTD itk_expat_XML_UseForeignDTD
 #define XML_UseParserAsHandlerArg itk_expat_XML_UseParserAsHandlerArg
+#define XmlGetUtf16InternalEncoding itk_expat_XmlGetUtf16InternalEncoding
```

Comment:
This got re-sorted here from line 50. Does [`export LC_COLLATE=POSIX`](https://github.com/InsightSoftwareConsortium/ITK/blob/v5.3rc01/Modules/ThirdParty/HDF5/src/itkhdf5/src/itk_hdf5_mangle.h#L17) give the same sort order as before? Or maybe [some other common choice](https://stackoverflow.com/questions/23660247/treatment-of-spaces-in-sort-command-difference-between-lc-collate-c-and-lc-coll)?

Reply:
Changing `LC_COLLATE` didn't change the output for me, but using `-d` and `-f` flags for sort seemed to do the job.
```
  -d, --dictionary-order      consider only blanks and alphanumeric characters
  -f, --ignore-case           fold lower case to upper case characters
```